lib/install/management/create.go

```diff
@@ -117,7 +116,7 @@ func (d *Dispatcher) uploadImages(files map[string]string) error {
 			defer wg.Done()
 
 			log.Infof("\t%q", image)
-			err = d.session.Datastore.UploadFile(d.ctx, image, path.Join(d.vmPathName, key), nil)
+			err := d.session.Datastore.UploadFile(d.ctx, image, path.Join(d.vmPathName, key), nil)
 			if err != nil {
```

Comment:
While you are here can you remove sending nil into the channel as well as you can remove check for nil while reading from results.